README.md

```diff
@@ -46,6 +46,19 @@ To import all the Sass rules from GOV.UK Prototype Components, add the following
 @import "govuk-prototype-components/x-govuk/all";
 ```
 
+> If you are using the GOV.UK Prototype Kit, replace the `loadPath` option for the Sass renderer with `includePaths` in `lib/build/tasks.js`:
+>
+> ```diff
+>   const result = sass.renderSync({
+>     file: `${sassPath}/${file}`,
+>     quietDeps: true,
+> -   loadPaths: [__dirname],
+> +   includePaths: [__dirname, 'node_modules'],
```

Comment:
I wonder if this could be recommended in the kit, using node_modules is a pretty common need.